vtex/address-form
# Add address rules for NIC and SLV

#### What is the purpose of this pull request?

Includes address rules for El Salvador (SLV) and Nicaragua (NIC) according to their state/city relationship.

The mapping was provided by the CX team (Mexico) together with the local stores via ticket [#230225](https://vtexhelp.zendesk.com/agent/tickets/230225) (Zendesk).

#### How should this be manually tested?

You can use the cart below to check the address-form both via Shipping Preview or Omnishipping for the updated regions.

- https://address--vtexgame1.myvtex.com/checkout/cart/add?sku=312&qty=1&seller=1&redirect=true&sc=2

#### Types of changes
